ext/opentelemetry-ext-asyncpg/setup.cfg

```diff
@@ -39,7 +39,7 @@ package_dir=
     =src
 packages=find_namespace:
 install_requires =
-    opentelemetry-api == 0.9.dev0
+    opentelemetry-api == 0.9b0
```

Comment:
```suggestion
    opentelemetry-api == 0.10.dev0
    wrapt >= 1.0.0, < 2.0.0
```

Reply:
fix ready

also bumped __version__ in version.py, is this correct?